controllers/PasswordController.js

```diff
@@ -36,7 +36,7 @@ class PasswordResetController {
       const emailBody = `
      <div>
          <h1> please follow this link to update your password </h1>
-         ${token}
+         https://heimdal-frontend.herokuapp.com/updatepassword/${token}
```

Comment:
Will this link be clickable in the user email?

Reply:
yes I will be clickable, your browser knows how to identify links